staging/src/k8s.io/kubectl/pkg/explain/v2/template.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"embed"
+	"path/filepath"
+	"strings"
+)
+
+//go:embed templates
+var rawBuiltinTemplates embed.FS
+
+func registerBuiltinTemplates(gen *generator) error {
```

Comment:
Should be easy to test? Make sure you have the right templates registered? That sounds a little bit like a "change detector",  but I guess it still tests that the loading works properly.

Reply:
added unit tests